pkg/registry/batch/job/strategy_test.go

```diff
@@ -153,21 +153,10 @@ func testJobStrategy(t *testing.T) {
 		t.Errorf("Expected a validation error")
 	}
 
-	// Existing gated fields should be preserved
```

Comment:
why was this removed?

Reply:
To keep the test simple. Other fields are already tested above.
It would be ideal to have a test table so that we don't have to repeat the checks, but kind of late to do such refactoring.

Reply:
I don't know if removing the TTL test is a good idea though

Reply:
> I don't know if removing the TTL test is a good idea though

/hold
just saw this... agree that this should not change unrelated fields

Reply:
changes in TTL is already tested above

Reply:
ack

/hold cancel

Reply:
not for PrepareForUpdate though

Reply:
https://github.com/kubernetes/kubernetes/blob/b345913c5eb794ded1a577db66e9f566eb8e5957/pkg/registry/batch/job/strategy_test.go#L134